opm/common/utility/numeric/calc_cellvol.hpp

```diff
@@ -0,0 +1,8 @@
+#include <vector>
+#include <math.h>  
+
+//using namespace std;
```

Comment:
Remove out-commented code.

Also, the files are missing a copyright header stating that it is under the GPL etc., see for example UniformTableLinear.hpp.